src/client.js

```diff
@@ -296,6 +296,8 @@ class ConnectionParameters {
             }    
         }
         this._options._storage = storage;
+        // Refresh client Callback
```

Comment:
Not sure this comment is really usefull.

Reply:
done